cli/lsp/analysis.rs

```diff
@@ -249,12 +250,20 @@ impl<'a> TsResponseImportMapper<'a> {
     &self,
     specifier: &ModuleSpecifier,
   ) -> Option<String> {
-    let specifier_path = specifier.to_file_path().ok()?;
-    let root_folder = self
-      .npm_resolver
-      .as_ref()
-      .and_then(|r| r.resolve_package_folder_from_path(specifier).ok())
-      .flatten()?;
+    fn closest_package_json_parent_dir(file_path: &Path) -> Option<&Path> {
```

Comment:
Could `find_closes_package_json` be reused here somehow?

Reply:
Yup, updated.